apps/win32/nuget/Microsoft.FluentUI.FluentTesterWin32.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.FluentUI.FluentTesterWin32</id>
+    <version>1.0.0</version>
```

Comment:
We have a few scripts that set the version number to "Date-<number of publishes today>" that you might be able to reuse